Conjure is exiting due to user errors.
Error:
    tests/parse_print/random_perturb/set01/change-07/change-07.essence line 3, column 29:
unexpected {
expecting ) or expression
    line 3, column 29
    find x : set (size 2) of int({..3)
                             ^^^^^^^